pipelines/mnist-pipelines/README.md

```diff
@@ -1,5 +1,7 @@
 # MNIST Pipelines GCP
 
+**This example does not currently work properly.  It will be updated soon.**
```

Comment:
Suggestion: Replace "It will be updated soon." with something like:

```
For more information, see [kubeflow/website #1611](https://github.com/kubeflow/website/issues/1611) and [kubeflow/pipelines #3037](https://github.com/kubeflow/pipelines/issues/3037).
```